benchmark/README.md

```diff
@@ -14,9 +14,21 @@ This folder contains benchmarks to measure the performance of the Node.js APIs.
 
 ## Prerequisites
 
-Most of the http benchmarks require [`wrk`][wrk] to be installed. It may be
-available through your preferred package manager. If not, `wrk` can be built
-[from source][wrk] via `make`.
+Most of the HTTP benchmarks require a benchmarker to be installed, this can be
+either [`wrk`][wrk] or [`autocannon`][autocannon].
+
+`Autocannon` is a Node script that can be installed using
+`npm install -g autocannon`. It will use the Node executable that is in the
+path, hence if you want to compare two HTTP benchmark runs make sure that the
+Node version in the path is not altered.
+
+`wrk` may be available through your preferred package manger. If not, you can
+easily build it [from source][wrk] via `make`.
+
+By default first found benchmark tool will be used to run HTTP benchmarks. You
+can overridde this by seting `NODE_HTTP_BENCHMARKER` environment variable to
```

Comment:
I can see it's usefulness, but do we really need it. I would like to avoid adding unnecessary environment flags. This could also be accomplished by using `--set benchmarker=autocannon`, which wouldn't require any extra code.

_I would appreciate other opinions._